### PR TITLE
Angels page #36

### PR DESCRIPTION
**Pull request by olavea**
Give me more tiny tasks 😸 👍

### CODE REVIEW

File: src/pages/angels.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react"
+import BrandLayout from "../components/BrandLayout"
+import SEO from "../components/seo"
+import { Button } from "@material-ui/core"
+
+const AngelsPage = () => {
+  const stripe = window.Stripe("pk_test_XiirziufElakjoOpyuyCrPfo")
+  const [sku, setSku] = useState("sku_G7M4igcIzjEhU2")
```
Comment (raae) [inline src/pages/angels.js]: You do not need this, as you never use setSku. It was probably used to do something with dropdown in the tutorial you watched.

File: src/pages/angels.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      cancelUrl: "http://localhost:8000/cancel",
+    })
+  }
+  return (
+    <BrandLayout>
+      <SEO title="Home" />
+      <h1>POW!</h1>
+      <div>
+        <div>
+          <Button
```
Comment (raae) [inline src/pages/angels.js]: Delete this button.

File: src/pages/angels.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          quantity: 1,
+        },
+      ],
+      successUrl: "http://localhost:8000/success",
+      cancelUrl: "http://localhost:8000/cancel",
+    })
+  }
+  return (
+    <BrandLayout>
+      <SEO title="Home" />
```
Comment (raae) [inline src/pages/angels.js]: The title should be "Angels"

**Comment by raae** [comment]
All in all very good solution, very clean :D

File: src/pages/angels.js (hunk 1 above)
Comment (olavea) [reply src/pages/angels.js]: You mean I don't need 
`const [sku, setSku] = useState("sku_G7M4igcIzjEhU2")`

File: src/pages/angels.js (hunk 3 above)
Comment (olavea) [reply src/pages/angels.js]: Ok

File: src/pages/angels.js (hunk 2 above)
Comment (olavea) [reply src/pages/angels.js]: Delete the BRONZE button?

Comment (raae) [reply src/pages/angels.js]: You have two bronze buttons.

File: src/pages/angels.js [hunk 4]
```diff
@@ -1,11 +1,17 @@
-import React, { useState, useEffect } from "react"
+import React from "react"
 import BrandLayout from "../components/BrandLayout"
 import SEO from "../components/seo"
 import { Button } from "@material-ui/core"
 
 const AngelsPage = () => {
-  const stripe = window.Stripe("pk_test_XiirziufElakjoOpyuyCrPfo")
-  const [sku, setSku] = useState("sku_G7M4igcIzjEhU2")
+  const STRIPE_KEY = "pk_test_XiirziufElakjoOpyuyCrPfo"
```
Comment (raae) [inline src/pages/angels.js]: This is not technically wrong, but we usually define them right under the imports.

**Comment by olavea** [comment]
Take a look

File: src/pages/angels.js [hunk 5]
```diff
@@ -31,41 +36,47 @@ const AngelsPage = () => {
...
         <div>
-          <Button
-            onClick={() => placeOrder(BRONZE_SKU)}
-            variant="contained"
-            color="primary"
-          >
-            BRONZE
-          </Button>
+          <AngelCard
+            metalText="Bronze"
```
Comment (raae) [inline src/pages/angels.js]: I would change `metalText` to `title`.

File: src/pages/angels.js [hunk 6]
```diff
@@ -31,41 +36,47 @@ const AngelsPage = () => {
...
+            Pricetext="500 NOK"
```
Comment (raae) [inline src/pages/angels.js]: Always use camelCase, so I would change `Pricetext` to `priceText`.

File: src/pages/angels.js [hunk 7]
```diff
@@ -31,41 +36,47 @@ const AngelsPage = () => {
...
+            spoText="15 spots"
```
Comment (raae) [inline src/pages/angels.js]: Change `spoText` to `spotsText`. Very seldom should you shorten words.

File: src/pages/angels.js [hunk 8]
```diff
@@ -31,41 +36,47 @@ const AngelsPage = () => {
...
+            liText="Lifetime access to POW! for one person."
```
Comment (raae) [inline src/pages/angels.js]: Change `liText` to `description`.

File: src/components/AngelCard.js [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+  },
+  pos: {
+    marginBottom: 12,
+  },
+  list: {
+    paddingLeft: "1.5em",
+  },
+})
+
+export default function AngelCard(props) {
```
Comment (raae) [inline src/components/AngelCard.js]: I like this style `const AngelCard = (props) => {` then at the bottom `export default AngelCard`

Comment (olavea) [reply src/components/AngelCard.js]: 👍

File: src/pages/angels.js (hunk 8 above)
Comment (olavea) [reply src/pages/angels.js]: 👍

File: src/pages/angels.js (hunk 7 above)
Comment (olavea) [reply src/pages/angels.js]: 👍

File: src/pages/angels.js (hunk 6 above)
Comment (olavea) [reply src/pages/angels.js]: 👍

File: src/pages/angels.js (hunk 5 above)
Comment (olavea) [reply src/pages/angels.js]: 😸

**Comment by olavea** [comment]
I made changes in layout of AngelCard.js component, see screenshot here
1_year_of_POW-I_Chose_screenshot_for_Pull_Req-33
this commit: 9c50493

**Comment by olavea** [comment]
Ok @raae how do I move this forward? I think I have done what I should.

File: src/pages/angels.js [hunk 10]
```diff
@@ -39,17 +35,10 @@ const AngelsPage = () => {
...
-          <AngelCard
-            weddingAnniversary="Bronze"
-            priceText="800 NOK"
+            weddingAnniversary="Bronze Angel"
+            priceText="500 NOK"
             spotsText="15 spots"
-            description="Lifetime access to POW! for one person."
+            description1="Lifetime access to POW! for one person."
             buttonText="Select"
             skuProp={BRONZE_SKU}
```
Comment (raae) [inline src/pages/angels.js]: You do not need `skuProp` on AngelCard, they are not used.

File: src/components/AngelCard.js [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+          <li>{props.description1}</li>
+          <li>{props.description2}</li>
+          <li>{props.description3}</li>
+        </Typography>
+      </CardContent>
+      <CardActions>
+        <Button
+          variant="contained"
+          color="primary"
+          skuPropProp={props.skuProp}
```
Comment (raae) [inline src/components/AngelCard.js]: Material buttons do not except `skuPropProp` as a prop.

File: src/components/AngelCard.js [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+          buttonTextPropProp={props.buttonText}
```
Comment (raae) [inline src/components/AngelCard.js]: Material buttons do not except `buttonTextPropProp` as a prop.

Its also goo to never name a prop, something with prop.

File: src/components/AngelCard.js [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+          POW!
+        </Typography>
+        <Typography variant="h5" component="h2">
+          {props.weddingAnniversary}
+        </Typography>
+        <Typography className={classes.pos} color="textSecondary">
+          {props.priceText} {bull} {props.spotsText}
+        </Typography>
+        <Typography className={classes.list} variant="body2" component="ul">
+          <li>{props.description1}</li>
```
Comment (raae) [inline src/components/AngelCard.js]: You need to conditionally render these.

`{props.description1 && <li>{props.description1}</li>}`

File: src/components/AngelCard.js [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+      </CardContent>
+      <CardActions>
+        <Button
+          variant="contained"
+          color="primary"
+          skuPropProp={props.skuProp}
+          buttonTextPropProp={props.buttonText}
+          onClick={props.onClick}
+        >
+          {props.buttonText}{" "}
```
Comment (raae) [inline src/components/AngelCard.js]: {" "} adds a space character and is not needed.

File: src/pages/angels.js (hunk 10 above)
Comment (olavea) [reply src/pages/angels.js]: Check

File: src/components/AngelCard.js (hunk 12 above)
Comment (olavea) [reply src/components/AngelCard.js]: Check

File: src/components/AngelCard.js (hunk 13 above)
Comment (olavea) [reply src/components/AngelCard.js]: Not anymore because of the .map thing right? @raae

File: src/components/AngelCard.js (hunk 14 above)
Comment (olavea) [reply src/components/AngelCard.js]: Ok I removed the space character (👽).

Comment (olavea) [reply src/components/AngelCard.js]: 58d3650

File: src/components/AngelCard.js (hunk 13 above)
Comment (raae) [reply src/components/AngelCard.js]: That is correct :D

**Comment by olavea** [comment]
Yohoooo! 😸  👍

**Pull request merged**